### PR TITLE
Enhance minitest matcher

### PR DESCRIPTION
**Pull request by TheSmartnik**
Hi,

I've added support for broadcasting to objects in minitest as was discussed in https://github.com/palkan/action-cable-testing/issues/11. Hope it's alright 😄

### CODE REVIEW

File: lib/action_cable/test_helper.rb [hunk 1]
```diff
@@ -127,5 +131,21 @@ def pubsub_adapter # :nodoc:
       def broadcasts_size(channel) # :nodoc:
         broadcasts(channel).size
       end
+
+      def stream(target, channel = nil)
+        return target if target.is_a?(String)
+
+        channel ||= @subscription
+        if channel && channel.respond_to?(:channel_name)
```
Comment (palkan) [inline lib/action_cable/test_helper.rb]: Let's avoid additional _raising_ method and write smth like:

```ruby
...
raise CHANNEL_NOT_FOUND unless channel && channel.respond_to?(:channel_name)
channel.broadcasting_for(...)
```

And move the message itself into a constant.

File: README.md [hunk 2]
```diff
@@ -121,6 +121,34 @@ class ChatChannelTest < ActionCable::Channel::TestCase
...
+  def test_broadcating
+    assert_broadcasts(@room, 1) do
+      perform :speak, message: "I'm here!"
+    end
+  end
+
+  # or
+
+  def test_broadcasted_data
+    assert_broadcasts_on(@room, text: "I'm here!", from: "John") do
```
Comment (palkan) [inline README.md]: Let's add an example with explicitly set `channel`.

And also we need to update `features/Minitest.md`

**Comment by TheSmartnik** [comment]
@palkan thanks for your feedback.
I've updated docs and fixed broken link in `features/README`

File: lib/action_cable/test_helper.rb (hunk 1 above)
Comment (TheSmartnik) [reply lib/action_cable/test_helper.rb]: @palkan I've created separate error class for this. Not sure if that's what you were asking, though

File: features/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@ This gem provides missing testing utils for [Action Cable][].
 
 **NOTE:** this gem is just a combination of two PRs to Rails itself ([#23211](https://github.com/rails/rails/pull/23211) and [#27191](https://github.com/rails/rails/pull/27191)) and (hopefully) will be merged into Rails eventually.
 
-**NOTE 2:** this is the documentation for (mostly) RSpec part of the gem. For Minitest usage see the repo's [Readme](https://github.com/palkan/action-cable-testing) or [Minitest Usage](minitest) chapter.
+**NOTE 2:** this is the documentation for (mostly) RSpec part of the gem. For Minitest usage see the repo's [Readme](https://github.com/palkan/action-cable-testing) or [Minitest Usage](Minitest.md) chapter.
```
Comment (palkan) [inline features/README.md]: The link was correct; see https://relishapp.com/palkan/action-cable-testing/docs/minitest

**Comment by palkan** [comment]
Please, add a Changelog entry

File: features/README.md (hunk 3 above)
Comment (TheSmartnik) [reply features/README.md]: @palkan Oh I see, thanks. I javen't thought about relishapp

**Comment by palkan** [comment]
Released as 0.2.0.

Thanks!

**Pull request merged**